docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,257 @@
+    * Georeference: coordinate for the top left pixel , the size of each pixel in the X direction, the size of each pixel in the Y direction, and the amount (if any) by which the product is rotated.
+
+
+## What data do I need?
+
+What to consider when chosing data:
+
+* Sensor
+    * Optical, Multispectal: [Landsat](https://landsat.gsfc.nasa.gov/), [MODIS](https://modis.gsfc.nasa.gov/), [Copernicus Sentinel-2](https://sentinel.esa.int/web/sentinel/missions/sentinel-2),[Pleiades](https://pleiades.cnes.fr/en/PLEIADES/index.htm), [Spot](https://www.intelligence-airbusds.com/imagery/constellation/spot/), [Planet](https://www.planet.com/products/planet-imagery/), [WorldView](http://worldview3.digitalglobe.com/), [Proba-V](https://earth.esa.int/eogateway/catalog/proba-v-1km-333m-and-100m-products), ...
+    * RADAR (Radio Detection and Ranging) -> SAR (Synthetic Aperture Radar): [Copernicus Sentinel 1](https://sentinel.esa.int/web/sentinel/missions/sentinel-1), [Radarsat](https://www.asc-csa.gc.ca/eng/satellites/radarsat/), [TanDEM-X](https://www.dlr.de/content/en/missions/tandem-x.html), [TerraSAR-X](https://www.dlr.de/content/en/missions/terrasar-x.html), [ICEYE](https://www.iceye.com/),...
+    * LiDAR (Light Detection and Ranging): [ICESat](https://icesat-2.gsfc.nasa.gov/), [GEDI](https://gedi.umd.edu/), ...
+* Resolution
+    * Temporal: when and how often a certain area is visited
+    * Spatial: the area on the ground that each pixel covers
+    * Spectral: the area of the electromagnetic spectrum that is observed and spectral width of each band provided
+        * Depending on the wavelengths observed,  clouds and atmospheric artifacts may result in data gaps
+    * Radiometric: how many values are possible for each pixel (bit-depth)
+* Costs
+    * Free: e.g. Landsat, MODIS, Sentinel, ...
+    * Non free (but might be possible to get for free/less for research): e.g. WorldView, Spot, Planet, ...
+* Preprocessing level
+    * Raw data
+    * Analysis ready data 
+    * Mosaics
+* User experience and knowledge
+    * RADAR/LiDAR require solid background knowledge for processing and interpretation
+    * Optical data is more easily interpreted and processed (and more pre-processed data is available)
+
+!!! default "EO database"
+
+    Database of all current and future EO missions and instrument information can be found in the [CEOS EO handbook database](http://database.eohandbook.com/database/instrumenttable.aspx)
+
+## Where can I find the data?
+
+The best place to get the data depends on your needs: Do you want to download the data into your own processing environment or do you need a processing environment close to the data? 
+
+!!! default "Other geospatial datasets"
+
+    For other geospatial vector and raster datasets, check out our [spatial dataset list on CSC research pages](https://research.csc.fi/open-gis-data#intdata3).
+
+### Services providing EO data and processing capabilities
+
+**CSC services**
+
```

Comment:
For Finnish researchers CSC computing services provide free of charge HPC and cloud resources, with wide selection of available [EO tools in Puhti HPC](#what-applications-are-available-on-puhti). Locally available are only a few Finnish EO datasets.